### PR TITLE
Add support for reel_share type

### PR DESCRIPTION
**Pull request by cce**
Addresses #35. Cool project @ifedapoolarewaju!

### CODE REVIEW

**Comment by ifedapoolarewaju** [comment]
wow, thank you so much for working on this!

File: browser/js/renderers.js [hunk 1]
```diff
@@ -69,6 +70,24 @@ function renderPost (post) {
   showInViewer(postDom);
 }
 
+function renderMessageAsReelShare (container, message) {
+  if (message._params.reelShare.media.video_versions) {
+    var url = typeof message === 'string' ? message : message._params.reelShare.media.video_versions[0].url
+    var img = dom(`<video controls src="${url}">`);
```
Comment (ifedapoolarewaju) [inline browser/js/renderers.js]: we should probably only display the image of the video within the chat message bubble, and then when the bubble is clicked, we show the video as you are already doing on line 82 🙂 

Or if we want to make the video playable within the message bubble, then we need to style the message bubble to contain it. Right now, it breaks out of the message bubble. 

screen shot 2017-10-21 at 5 29 03 pm

File: browser/js/renderers.js [hunk 2]
```diff
@@ -20,7 +21,7 @@ function renderMessage (message, direction, time, type) {
   if (!type && typeof message === 'string') type = 'text';
 
   if (renderers[type]) renderers[type](divContent, message);
-  else renderMessageAsText(divContent, '<unsupported message format>', true);
+  else renderMessageAsText(divContent, '<unsupported message format type '+type+'> '+JSON.stringify(message), true);
```
[the remark this answers is not in the document]
Comment (cce) [reply browser/js/renderers.js]: Yes, you're right, it was something I was using for debugging — I can put it back to normal.

[the remark this answers is not in the document]
Comment (ifedapoolarewaju) [reply browser/js/renderers.js]: >  I can put it back to normal

That would be cool, yes 🙂

**Pull request merged**